python/mxnet/optimizer.py

```diff
@@ -356,6 +358,52 @@ def update(self, index, weight, grad, state):
                        lr=lr, wd=wd, **kwargs)
 
 @register
+class MultiPrecisionSGD(Optimizer):
```

Comment:
C2 will have something like this, but they are discussing now the proper API (it is not in the PR form yet).